### PR TITLE
fix: duplicate notify if reviewer is requester

### PR DESCRIPTION
**Pull request by diartyz**

### CODE REVIEW

File: src/github.js [hunk 1]
```diff
@@ -91,16 +92,11 @@ exports.handleGitHubHook = (req, data) => {
...
-                sendAsBot(botToken, reviewerUserID, text),
-              ])
+              return sendAsBot(botToken, requesterUserID, text)
             } else if (requesterUserID) {
               return sendAsBot(botToken, requesterUserID, text + linkNotify(reviewerGitHubName))
-            } else if (reviewerUserID) {
-              return sendAsBot(botToken, reviewerUserID, text + linkNotify(requesterGitHubName))
             } else {
-              return (`Could not find users for ${requesterGitHubName} and ${reviewerGitHubName}`)
+              return (`Could not find users for ${requesterGitHubName}`)
```
Comment (EnixCoda) [inline src/github.js]:
```suggestion
              return (`Could not find user "${requesterGitHubName}"`)
```

Comment (diartyz) [reply src/github.js]: nono

Comment (EnixCoda) [reply src/github.js]: nope

Comment (diartyz) [reply src/github.js]: test

Comment (diartyz) [reply src/github.js]: test

**Comment by EnixCoda** [comment]
Scene: A drafts a PR and asked review from B.
* if B reviewed with review comments, A should be notified on Slack.
* do nothing for replies under review comments, bc 1. logic about it is complicated 2. no enough data provided from GitHub messages
* if B comments (like this one), B should not receive messages on Slack about this comment while A should.
* if A comments, neither A or B would receive messages.

### further
* handle PR approved/merged messages

File: src/github.js [hunk 2]
```diff
@@ -91,14 +92,11 @@ exports.handleGitHubHook = (req, data) => {
...
-              return Promise.all([
-                sendAsBot(botToken, requesterUserID, text),
-                sendAsBot(botToken, reviewerUserID, text),
-              ])
+              return sendAsBot(botToken, requesterUserID, text)
             } else if (requesterUserID) {
               return sendAsBot(botToken, requesterUserID, text + linkNotify(reviewerGitHubName))
             } else if (reviewerUserID) {
-              return sendAsBot(botToken, reviewerUserID, text + linkNotify(requesterGitHubName))
+              return sendAsBot(botToken, reviewerUserID, linkNotify(requesterGitHubName))
```
Comment (EnixCoda) [inline src/github.js]: 上面的 `text` 内容是针对 `requester` 的，不适用于这个情况？

Comment (diartyz) [reply src/github.js]: 不给评论者发信息

**Review by EnixCoda — APPROVED** [review]

**Pull request merged**